### PR TITLE
Add package installation steps in Readme

### PR DESCRIPTION
**Pull request by BushraAsif**

### CODE REVIEW

**Review by shahbaz-mehar — COMMENTED** [review]
Changes have been reviewed.

File: README.md [hunk 1]
```diff
@@ -11,9 +11,25 @@ This is an SDK for AltaPay written in JavaScript. It is written in a way that ma
...
+
+Install the package via npm
+
+``` npm i altapay-js-api ```
+
+### How to build
+
+Run below command to build the package
+
+> If you install the dependency using Method 2 then you have to run below command in `node_modules/altapay-js-api`
```
Comment (emicha) [inline README.md:32]: This should be part of the **Method 1** block right? Since build package is only used for including this as a dependency.

**Comment by maf-embraceit** [comment]
All changes have been reviewed.

**Comment by emicha** [comment]
The requirements are only necessary for the build step, which is confusing that is in the top, on a big heading
image

**Review by emicha — APPROVED** [review]

**Pull request merged**